app/forms/application_form.rb

```diff
@@ -15,10 +15,10 @@ class << self
     #   class UserForm < ApplicationForm
     #     form_attributes :first_name, :last_name
     #   end
-    # @param attributes [Array<Symbol>] - the attributes to be defined on the form
+    # @param attributes [Array<Symbol>] - the attributes available on the form
     # @return [void]
     def form_attributes(*attributes)
-      @attributes ||= attributes.map do |attribute|
+      @form_attributes ||= attributes.map do |attribute|
```

Comment:
does this impact any apps currently using the application_form?

Reply:
Simply a name change since Rubocop thinks a caching instance var should be called like the method that wraps it, which is actually reasonable. People shouldn't be accessing this directly, and even if they did it wouldn't be affected by a change here.